eng/common/pipelines/templates/steps/create-pull-request.yml

```diff
@@ -83,7 +83,7 @@ steps:
       -PRTitle "${{ parameters.PRTitle }}"
       -PRBody "${{ coalesce(parameters.PRBody, parameters.CommitMsg, parameters.PRTitle) }}"
       -PRLabels "${{ parameters.PRLabels }}"
-      -UserReviewers "${{ parameters.GHReviewersVariable }}"
-      -TeamReviewers "${{ parameters.GHTeamReviewersVariable }}"
+      -UserReviewers "$(${{ parameters.GHReviewersVariable }})"
```

Comment:
I _believe_ this should work. At queue time the parameter should resolve and become a standard Build Variable access.

Reply:
@weshaggard of course we need to test it! I would never merge any changes here without running the template pipelines.